projects/vdk-control-cli/src/vdk/internal/control/rest_lib/rest_client_errors.py

```diff
@@ -12,31 +12,38 @@
 
 
 class ApiClientErrorDecorator:
-    def __init__(self, what="Control Service Error"):
-        self.what = what
+    def __init__(
+        self, what="Control Service Error", consequences="Operation cannot complete."
+    ):
+        self.__what = what
+        self.__consequences = consequences
 
     def _get_error(self, exception: ApiException):
-        # noinspection PyBroadException
         try:
             return json.loads(exception.body)
         except:
             error = {}
             if exception.status == 401:
                 error = dict(
-                    what=self.what,
-                    why="The request has not been applied because it lacks valid authentication credentials.",
-                    consequences="Operation cannot complete.",
-                    countermeasures="Try to login again using VDK CLI login command. "
-                    "Make sure you have permission to execute the given operation.",
+                    what=self.__what,
+                    why=f"The request has not been applied because it lacks valid authentication credentials. "
+                    f"Error returned by control service is {exception.body}",
+                    consequences=self.__consequences,
+                    countermeasures="Try to login again using VDK CLI login command."
+                    "Or set correct api token configuration (see vdk config-help)."
+                    "Make sure you have permission to execute the given operation."
+                    "Investigate (google) the error returned by control service."
```

Comment:
nit: "(google)" seems a bit weird to include. Just "Investigate" is probably fine.

Reply:
It's honest advice. A lot of people don't google enough :) Hopefully, someone finds it funny as well.